db/migrate/20171003143034_add_comments_count_to_polls.rb

```diff
@@ -0,0 +1,7 @@
+class AddCommentsCountToPolls < ActiveRecord::Migration
+  def change
+    add_column :polls, :comments_count, :integer, default: 0
+    add_column :polls, :author_id, :integer
+    add_column :polls, :hidden_at, :datetime
```

Comment:
Please, remove these two lines:
```
add_column :polls, :author_id, :integer
add_column :polls, :hidden_at, :datetime
```

Reply:
Those two fields are required by comments model. When removing it from polls (and removing the columns from the migration), raises an error asking for either "hidden_at" or "author_id".

Reply:
Oh sorry @zamuro you are right!
There is probably a way to bypass this, but it's a situation we have accepted in the past.
Don't worry about it 🙏 👍